### PR TITLE
Feature/align to target regarding issue #3

### PR DESCRIPTION
**Pull request by miladkdz**

### CODE REVIEW

File: src/vaadin-tooltip.html [hunk 1]
```diff
@@ -134,25 +144,65 @@
...
+                  break;
+                default:
+                  tooltipLeft = targetLeft + horizontalCenterOffset;
+              }
               break;
             case 'bottom':
-              tooltipLeft = targetLeft + horizontalCenterOffset;
               tooltipTop = targetTop + targetRect.height;
+
+              switch (this.align) {
```
Comment (peancored) [inline src/vaadin-tooltip.html]: The parts of the code seem to be equal, so can be extracted to a function.

File: src/vaadin-tooltip.html [hunk 2]
```diff
@@ -134,25 +144,65 @@
...
+                  break;
+                default:
+                  tooltipTop = targetTop + verticalCenterOffset;
+              }
               break;
             case 'right':
               tooltipLeft = targetLeft + targetRect.width;
-              tooltipTop = targetTop + verticalCenterOffset;
+
+              switch (this.align) {
```
Comment (peancored) [inline src/vaadin-tooltip.html]: The parts of the code seem to be equal, so can be extracted to a function.

File: src/vaadin-tooltip.html (hunk 1 above)
Comment (miladkdz) [reply src/vaadin-tooltip.html]: Done.

File: src/vaadin-tooltip.html (hunk 2 above)
Comment (miladkdz) [reply src/vaadin-tooltip.html]: Done.

**Pull request merged**